<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1513</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_select">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sélectionner un répertoire source</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aucun répertoire source sélectionné</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_select_target">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sélectionner un répertoire destination</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_target">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Aucun répertoire destination sélectionné</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>900</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>170</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>25</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_quit">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>750</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCopyorMove">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>10</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Copie (sinon les fichiers sont déplacés)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalfichiers">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>fichiers restants</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Déplacer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGarder">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Garder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSupprimer">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>750</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRenommer">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Renommer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalrepertoires">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>710</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>710</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>repertoires restants</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>690</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Répertoire : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSuivant">
   <property name="geometry">
    <rect>
     <x>1452</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrecedent">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRepSource">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>681</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRepertoireValider">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>770</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Freeware</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1440</x>
     <y>710</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CurrentImage">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>710</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>780</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_TotalImage">
   <property name="geometry">
    <rect>
     <x>1470</x>
     <y>710</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
